pandas/core/generic.py

```diff
@@ -3137,6 +3137,78 @@ def clip_lower(self, threshold, axis=None):
         subset = self.ge(threshold, axis=axis) | isnull(self)
         return self.where(subset, threshold, axis=axis)
 
+    def round(self, decimals, out=None):
```

Comment:
Sadly numpy needs a positional argument for `out` with `np.round`. So I think we should stick with this signature, just removing it from the docstring